analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RuleChecks/EmptyCollectionsShouldNotBeEnumeratedBase.cs

```diff
@@ -138,6 +138,11 @@ protected override ProgramState PreProcessSimple(SymbolicContext context)
         return state;
     }
 
+    private static bool IsEmptyObject(IOperation operation) =>
```

Comment:
Because it looks at `TrackedCollectionTypes`
```suggestion
    private static bool IsEmptyCollection(IOperation operation) =>
```